src/components/bcros/businessDetails/index.vue

```diff
@@ -94,9 +94,13 @@ watch(() => route.params.identifier as string, loadComponentData, { immediate: t
 </script>
 
 <template>
-  <div id="bcros-business-details" class="bg-white h-[150px]" data-cy="business-details">
-    <div class="flex pt-5 text-bcGovGray-900 app-inner-container">
-      <div class="grow" data-cy="business-details-name">
+  <div
+    id="bcros-business-details"
```

Comment:
In the temp url, there are some extra space at the top of the tombstone. I guess it is caused by this 'pt-7'.